logbook-core/src/main/java/org/zalando/logbook/DefaultHttpLogFormatter.java

```diff
@@ -42,7 +43,12 @@ public String format(final Precorrelation<HttpRequest> precorrelation) throws IO
     }
 
     private String formatRequestLine(final HttpRequest request) {
-        return String.format("%s %s HTTP/1.1", request.getMethod(), request.getRequestUri());
+        return String.format("%s %s HTTP/1.1", request.getMethod(), renderRequestUri(request));
```

Comment:
Unrelated to this PR, but we don't actually know the request was done with HTTP/1.1, for a servlet implementation this string could be retrieved from `ServletRequest.getProtocol`.